VideoLocker/res/layout/edx_spinner_dropdown_item.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<CheckedTextView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@android:id/text1"
```

Comment:
Taking a quick look at the source of `ArrayAdapter`, I don't see this ID referenced anywhere. The `createViewFromResource()` method (which is called by both `getView()` and `getDropDownView()`) uses the base `TextView` if it's at the root; otherwise it uses the `textViewResourceId` constructor parameter to find the `TextView` (which is not provided in the constructor that you call).